wallets/rn_cli_wallet/src/screens/OnboardingScreen.tsx

```diff
@@ -0,0 +1,76 @@
+import React, {useEffect} from 'react';
+import {
+  StatusBar,
+  useColorScheme,
+  View,
+  StyleSheet,
+  ImageBackground,
+} from 'react-native';
+
+import {Colors} from 'react-native/Libraries/NewAppScreen';
+import '@walletconnect/react-native-compat';
+import {GetStartedButton} from '../components/GetStartedButton';
+import {W3WText} from '../components/W3WText';
+import {TextContent} from '../utils/Text';
+import {SafeAreaView} from 'react-native-safe-area-context';
+import useInitialization from '../hooks/useInitialization';
+
+const backgroundImageSrc = require('../assets/ethCalculatorBG.png');
+
+const OnboardingScreen = () => {
+  const initialized = useInitialization();
+
+  useEffect(() => {
+    console.log('Web3WalletSDK initialized:', initialized);
+  }, [initialized]);
+
+  const isDarkMode = useColorScheme() === 'dark';
+  const backgroundStyle = {
+    flex: 1,
+    backgroundColor: isDarkMode ? Colors.darker : Colors.lighter,
+  };
+
+  return (
+    <View style={{flex: 1, backgroundColor: 'white'}}>
```

Comment:
move the inline style to the `styles` object below. Same with other inline styles